common/djangoapps/student/views.py

```diff
@@ -1032,8 +1033,11 @@ def notify_enrollment_by_email(course, user, request):
 
         except Exception:
             log.error('unable to send course enrollment verification email to user from "{from_address}"'.format(
-                        from_address=from_address), exc_info = True)
-            return JsonResponse({"is_success": False, "error": _("Could not send enrollment email to the user"),})
+                        from_address=from_address),exc_info=True)
```

Comment:
Something looks off here.. It looks like a space was removed between kwargs `...from_address),exc_info` (no space after comma).

Let's clean up this entire `log` statement like:
```python
log.error(
    "unable to send course enrollment verification email to user from '{from_address}'".format(
        from_address=from_address,
    ),
    exc_info=True,
)
```